<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KGraphViewerPreferencesParsingWidget</class>
 <widget class="QWidget" name="KGraphViewerPreferencesParsingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>104</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="parsingMode">
     <property name="title">
      <string>Parse using:</string>
     </property>
     <property name="selectedId" stdset="0">
      <number>2</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="external">
        <property name="text">
         <string>E&amp;xternal command</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="internal">
        <property name="text">
         <string>Internal &amp;library</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
